app/controllers/concerns/reports/csv_export_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Reports
+
+  module CsvExportController
+
+    def export_all
+      generate_report_data_csv
+    end
+
+    private
+
+    def email_to_address
+      params[:email_to_address].presence || current_user.email
+    end
+
+    def generate_report_data_csv
+      ExportRawReportMailer.delay.raw_report_email(email_to_address, raw_report)
```

Comment:
I added a `TODO` comment. `git grep '\.delay\.'` shows a few more that could use `deliver_later` conversion.